<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissPutinDlg</class>
 <widget class="QDialog" name="MissPutinDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>318</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>入库</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="4">
    <widget class="QComboBox" name="combPdiType">
     <item>
      <property name="text">
       <string>正品</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>样品</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="lblPdiDesc">
     <property name="text">
      <string>备注:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="cboxPutinNow">
     <property name="text">
      <string>现在</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="lblPutInDate">
     <property name="text">
      <string>入库时间:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QDateTimeEdit" name="dtePutinDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblShelf">
     <property name="text">
      <string>货架号:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="edtShelf"/>
   </item>
   <item row="8" column="0" colspan="5">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>保存并关闭</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveAndConti">
       <property name="text">
        <string>保存并继续</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="4">
    <widget class="QComboBox" name="combReason">
     <item>
      <property name="text">
       <string>生产</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>归还</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="lblPdiType">
     <property name="text">
      <string>样品标记:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="lblReason">
     <property name="text">
      <string>入库原因:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblPiDesc">
     <property name="text">
      <string>产品描述:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPlainTextEdit" name="pedtPiDesc">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPlainTextEdit" name="pedtPdiDesc">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblProdName">
     <property name="text">
      <string>产品型号:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="edtProdName"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblSerial">
     <property name="text">
      <string>序列号:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="edtSerial"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="lblTips">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtProdName</tabstop>
  <tabstop>edtSerial</tabstop>
  <tabstop>edtShelf</tabstop>
  <tabstop>pedtPiDesc</tabstop>
  <tabstop>cboxPutinNow</tabstop>
  <tabstop>dtePutinDate</tabstop>
  <tabstop>combPdiType</tabstop>
  <tabstop>combReason</tabstop>
  <tabstop>pedtPdiDesc</tabstop>
  <tabstop>btnSaveAndConti</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cboxPutinNow</sender>
   <signal>clicked(bool)</signal>
   <receiver>dtePutinDate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSaveAndConti</sender>
   <signal>clicked()</signal>
   <receiver>MissPutinDlg</receiver>
   <slot>s_saveAndConti()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>MissPutinDlg</receiver>
   <slot>s_saveAndClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtSerial</sender>
   <signal>editingFinished()</signal>
   <receiver>MissPutinDlg</receiver>
   <slot>s_querySerial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtProdName</sender>
   <signal>editingFinished()</signal>
   <receiver>MissPutinDlg</receiver>
   <slot>s_queryProdName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>s_saveAndConti()</slot>
  <slot>s_saveAndClose()</slot>
  <slot>s_querySerial()</slot>
  <slot>s_queryProdName()</slot>
 </slots>
</ui>
